index.md

```diff
@@ -5,46 +5,53 @@
 * understand system administration essentials
 * get a practical taste of what sysadmins do in industry.
 
-Lectures are scheduled for **Tuesdays (Beginner) and Thursdays (Advanced) 
-from 8 to 9pm** the OCF Lab, located at **171 MLK** (MLK basement down the 
-hall from the student store). **Attendance is mandatory, with two allowed 
-unexcused absences.** Some other conflicts, such as exams, may be excused 
-with prior notice. Please only come to the lab on the day corresponding to 
-the section you are in in order to save seats for students enrolled in the 
-section being taught on that day.
-
-The primary assignment in this course will be **weekly lab work**.
-In keeping with the name of the course, labs will be significantly hands-on.
-You will be **working on real systems**, configuring and fixing things.
-**Lecture will double as office hours** for students to come in and ask
-questions about the lab work. This is a **2 unit DeCal**.
+The course will be taught in two sections: one for students with minimal
+prior background in GNU/Linux or system administration, and one for
+students with more background/experience in the same. The sections are
+titled Basic and Advanced, respectively.
 
 An important skill for system administrators is the **willingness to learn
 quickly** about new and unfamiliar technologies, so while we expect many
 students will have a CS background, **the only real prerequisite is a desire
-to learn**. We know potential students will have varied backgrounds in terms
-of CS and sysadmin experience, therefore, we will be teaching this course in
-two tracks: a beginner's track that will focus more on introducing new users
-to GNU/Linux, and an advanced track that will cater to students with more
-experience using Linux. Both tracks will get get started fast and move
-quickly - there's a lot of material to cover.
+to learn**. The Basic section has been designed to introduce new users to Linux,
+and the Advanced section has been designed to give more experienced users a
+taste of what Linux is capable of. 
 
 Another important skill to learn is patience and perseverence.
 Don't be discouraged if things don't work with the first time: the internet
 is held together with the programmatic equivalent of duct tape, and the job
 of a sysadmin is best described as applying new duct tape when the old stuff
 wears out, which happens a lot.
+
+
+### Administrivia
+
+Lectures are scheduled for **Tuesdays** for the Basic section and **Thursdays**
+for the Advanced section, from from **8 to 9pm** the OCF Lab, located at
+**171 MLK** (MLK basement down the hall from the student store).
+**Attendance is mandatory, with two allowed unexcused absences.** Some other
+conflicts, such as exams, may be excused with prior notice. Please only come
+to the lab on the day corresponding to the section you are in in order to save
+seats for students enrolled in the section being taught on that day. 
```

Comment:
This last sentence is a bit confusing, are we telling students not to visit the lectures of the other class?

Reply:
Yeah I think this is confusing to read too, I'd just say "Please only come to the lab on the day corresponding to the section you are in." and leave it at that, I don't think we need to give a justification for it.